app/views/journal_feeds/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<body class="vsc-initialized">
+<main role="main" class="container">
+  <div class="row row-offcanvas row-offcanvas-right">
+    <div class="col-12 col-md-9">
+      <h2>Abstracts</h2>
+        <% @journal_feeds.each do |journal_feed| %>
+          <% current_user.subscriptions.each do |subscription| %>
+            <% if subscription.user_id == current_user.id %>
+              <% journal_feed.journals.each do |journal| %>
+                <% journal.abstracts.each do |abstract| %>
```

Comment:
You're doing a bit more work here than you need to. You're looping through all the joiurnal feeds and checking if the user is subscribed, but you could instead just loop through the user's subscriptions and pull the journal_feed out for each one. `current_user.subscriptions` has everything you need. We can dig into this tonight if it's not clear.